awx/main/migrations/0078_v360_clear_sessions_tokens_jt.py

```diff
@@ -19,11 +19,11 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='systemjob',
             name='job_type',
-            field=models.CharField(blank=True, choices=[('cleanup_jobs', 'Remove jobs older than a certain number of days'), ('cleanup_activitystream', 'Remove activity stream entries older than a certain number of days'), ('clearsessions', 'Removes expired browser sessions from the database'), ('cleartokens', 'Removes expired OAuth 2 access tokens and refresh tokens')], default='', max_length=32),
+            field=models.CharField(blank=True, choices=[('cleanup_jobs', 'Remove jobs older than a certain number of days'), ('cleanup_activitystream', 'Remove activity stream entries older than a certain number of days'), ('cleanup_sessions', 'Removes expired browser sessions from the database'), ('cleanup_tokens', 'Removes expired OAuth 2 access tokens and refresh tokens')], default='', max_length=32),
```

Comment:
@AlanCoding @rooftopcellist this needs changing, too (and the corresponding `models/jobs.py` change).

These choices don't actually match the `job_type` being set in the database.  The use of `choices=` here doesn't actually enforce this at the database level via e.g., an enum.  It's just being used to provide incorrect choices in the `OPTIONS` text.